src/components/app.tsx

```diff
@@ -23,7 +23,7 @@ import { IPortalData, IPortalDataUnion } from "../portal-types";
 import { signInWithToken, initializeDB, setPortalData, initializeAnonymousDB,
          onFirestoreSaveTimeout, onFirestoreSaveAfterTimeout, getPortalData } from "../firebase-db";
 import { Activity, IEmbeddablePlugin, Sequence } from "../types";
-import { initializeLara, LaraGlobalType } from "../lara-plugin/index";
+import { initializeLara, LaraGlobalType, PluginAPI_V3 } from "../lara-plugin/index";
```

Comment:
It might be better to directly import the `__closeAllPopups()` call so that it doesn't look like it is part of the API, something like:

`import { __closeAllPopups } from "../lara-plugin/plugin-api/popup.ts";`